### PR TITLE
[LD-817] Addition of params to mixed_people/search

### PR DESCRIPTION
**Pull request by Kavya-24**
RCE: https://www.notion.so/apolloio/RCE-API-Documentation-People-APIs-4507e2c99cdd460283978b8655635ef0
Jira: https://apollopde.atlassian.net/browse/LD-817
Slack Discussions: https://apolloio.slack.com/archives/C0616JV62QM/p1697527904956669

Before: 
Screenshot 2023-10-17 at 11 01 38 PM
Screenshot 2023-10-17 at 11 01 49 PM

After:
Screenshot 2023-10-23 at 4 14 00 PM
Screenshot 2023-10-23 at 4 14 08 PM
Screenshot 2023-10-18 at 12 01 50 AM

Make sure you've checked off all these things before submitting:
- [x] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [x] This pull request is submitted to the `master` branch.
- [x] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

**Comment by PavanSankavar** [comment]
is 'After' screenshot is for 'python' tab ?

File: source/includes/_people.md [hunk 1]
```diff
@@ -54,6 +62,30 @@ print(response.text)
             "value": "engineer manager",
             "display_name": "engineer manager"
         },
+        {
+            "label": "# Employees",
+            "signal_field_name": "organization_num_employees_ranges",
+            "value": "1,1000000",
+            "display_name": "1-1000000"
+        },
+        {
```
[the remark this answers is not in the document]
Comment (Kavya-24) [reply source/includes/_people.md:71]: |  rest of details are same for both

What do we mean by both?

File: source/includes/_people.md [hunk 2]
```diff
@@ -256,9 +289,19 @@ This endpoint searches for people and do not return any email information. To ge
...
+person_seniorities (optional)           | An array of strings denoting the seniorities or levels  | ["senior", "manager"]
+contact_email_status (optional)         | An array of strings to look for people having a set of email statuses | ["verified", "guessed", "unavailable", "bounced", "pending_manual_fulfillment"]
+q_organization_domains (optional)       | An array of the company domains to search for, joined by the new line character.  | "google.com\nfacebook.com"
+organization_locations (optional)       | An array of strings denoting allowed locations of organization headquarters of the person | ["California, US", "Minnesota, US"]
+organization_ids (optional)             | An array of organization ids obtained from companies-search | ["63ff0bc1ff57ba0001e7eXXX"]
+organization_num_employees_ranges (optional)   | An array of intervals to include people belonging in an organization having number of employees in a range | ["1,10", "101,200" ]
 page (optional) | An integer that allows you to paginate through the results  | 1
+per_page (optional) | An integer to load per_page results on a page. Should be in inclusive range [1, 100] | 10
```
Comment (atul-tw) [inline source/includes/_people.md:303]: We are exposing this field which can be easily misused by giving big numbers here. I did not find the validation on the limit of value for per_page. Should we expose this or expose and add validations in backend if it is not present?

Basically, we should not allow the customer to get people more than 100 in a single page

**Review by atul-tw — COMMENTED** [review]
Before exposing `per_page` please do a double check that we have validation that per_page cannot be more than 100

**Comment by Kavya-24** [comment]
> is 'After' screenshot is for 'python' tab ?

Yes. Added more screenshots of after for clarity.

File: source/includes/_people.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kavya-24) [reply source/includes/_people.md:71]: Need clarification on this @PavanSankavar

File: source/includes/_people.md (hunk 2 above)
Comment (Kavya-24) [reply source/includes/_people.md:303]: We are validating in two places.
1. For endpoints that do not require API Key, we check per-page [here](https://github.com/apolloio/leadgenie/blob/master/lib/rate_limiter.rb/#L368). The default max is 100
2. For APIs using es-searcher and supporting per page, we check in es_searcher validations [here](url)
Screenshot 2023-10-23 at 5 19 55 PM

Here, if we have id mode then the limit is 200 else 100. Verifed that for this endpoint, this validation is being conducted. @atul-tw

**Comment by Kavya-24** [comment]
> Before exposing `per_page` please do a double check that we have validation that per_page cannot be more than 100
Validated. Added findings [here](https://github.com/apolloio/apollo-api-docs/pull/60#discussion_r1368540656)
Screenshot 2023-10-23 at 5 22 07 PM

File: source/includes/_people.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PavanSankavar) [reply source/includes/_people.md:71]: we have two labels "Management Level", both with different display names, is this intentional ?

**Review by PavanSankavar — APPROVED** [review]
LGTM ! Need clarification on one comment

File: source/includes/_people.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kavya-24) [reply source/includes/_people.md:71]: This is auto generated from breadcrumbs itself. We are sending two person-seniorities (senior and manager) hence the labels are generated for both of them separately.

**Review by manickam-apl — APPROVED** [review]

**Comment by Kavya-24** [comment]
Screenshot 2023-10-26 at 5 19 27 PM

Merged.

**Pull request merged**